packages/tabs/src/Tabs.tsx

```diff
@@ -158,14 +153,14 @@ function Tabs({
 
     const enabledCurrentIndex = enabledIndexes.indexOf(selected!);
 
-    switch (e.key) {
-      case 'ArrowRight':
+    switch (e.keyCode) {
+      case keyMap['ArrowRight']:
```

Comment:
Why are we doing a string lookup?

Reply:
I thought it made it more readable, can eliminate if you think that makes more sense.

Reply:
🤔 Never knew that, thank you!